_data/compose-cli/docker_compose_restart.yaml

```diff
@@ -1,6 +1,16 @@
 command: docker compose restart
 short: Restart containers
-long: Restart containers
+long: |-
+  Restarts all stopped and running services.
```

Comment:
TIL it starts a stopped service as well 🤔 

(I assume it's not starting one-off containers started by `compose run`?)